uploadNeuroDB/minc_insertion.pl

```diff
@@ -145,44 +145,44 @@
 &Getopt::Tabular::SetHelp($Help, $Usage);
 &Getopt::Tabular::GetOptions(\@opt_table, \@ARGV) || exit 1;
 
-
-# input option error checking
-{ package Settings; do "$ENV{LORIS_CONFIG}/.loris_mri/$profile" }
-
-
-if ($profile && !@Settings::db) { 
-    print "\n\tERROR: You don't have a ".
-    "configuration file named '$profile' in:  $ENV{LORIS_CONFIG}/.loris_mri/ \n\n";
-    exit 2; 
+if(!$ENV{LORIS_CONFIG}) {
+	print STDERR "\n\tERROR: Environment variable 'LORIS_CONFIG' not set\n\n";
+	exit 1;  # TODO replace by appropriate exit code from ExitCodes once it is defined
```

Comment:
do you think it is worth it to change exit codes on the user given that they will be changed again in the Harmonize Exit codes PR?

Reply:
Actually, the numerical values used in the code that I corrected are in line with the intended usage of each code defined in `ExitCode.pm`:

```perl
our $PROFILE_FAILURE         = 2; # if no profile file specified
our $MISSING_ARG             = 3; # if missing script's argument(s)
our $DB_SETTINGS_FAILURE     = 4; # if DB settings in profile file are not set
our $ARG_FILE_DOES_NOT_EXIST = 5; # if file given as an argument does not exist in the file system
```
The only one I added was error code `1` for the missing environment variable. But we can add it in `ExitCode.pm`.

Reply:
@nicolasbrossard Now that the ExitCode has been merged, maybe you could use them in this PR?